Mathlib/Analysis/Normed/Lp/ProdLp.lean

```diff
@@ -458,6 +515,14 @@ theorem prod_continuous_equiv : Continuous (WithLp.equiv p (α × β)) :=
 theorem prod_continuous_equiv_symm : Continuous (WithLp.equiv p (α × β)).symm :=
   continuous_id
 
+@[continuity]
+theorem prod_continuous_toLp : Continuous (@WithLp.toLp p (α × β)) :=
+  continuous_id
+
+@[continuity]
+theorem prod_continuous_ofLp : Continuous (@WithLp.ofLp p (α × β)).symm :=
+  continuous_id
```

Comment:
`ofLp.symm` should be simplified to `toLp`, so no lemma should be written about it. Or maybe you meant
```suggestion
theorem prod_continuous_ofLp : Continuous (@WithLp.ofLp p (α × β)) :=
  continuous_id
```
?

Reply:
Oh yeah this was a typo!